serving/docs/lmi/tutorials/trtllm_finding_max_num_tokens_tutorial.md

```diff
@@ -0,0 +1,132 @@
+# TensorRT-LLM finding max_num_tokens tutorial
+
+From LMI 0.27.0 (TRTLLM 0.8.0), we introduced `max_num_tokens` build option to simplify the compilation of TRTLLM model as well as supporting all kinds of traffic pattern.
+This max number of tokens represent the shared tokens across the TRTLLM framework regardless of the batch size or context length.
+It is similar to pagedAttention tokens (e.g max_batched_tokens) settings in vLLM. However, this value is controlling the engine cache that has influence to the KVCache tokens itself.
+
+If you set this value in LMI, it is equivalent to the followings:
+
+```
+Serving prperties: option.max_num_tokens=50000
+ENV: OPTION_MAX_NUM_TOKENS=50000
+```
+
+It is the same as:
+
+```
+option.max_num_tokens=50000
+option.max_input_len=4096
+option.max_output_len=4096
+option.max_rolling_batch_size=256
```

Comment:
I think we need to call out that these configs are independent of the max_num_token configs. 

These configs are purely used for validation/generation limits/batch size restriction, and do not mean the server can handle 256 batch with 4096 total tokens each

Reply:
addressed with a comments below